graphql-client/src/Data/GraphQL/Error.hs

```diff
@@ -30,13 +33,35 @@ data GraphQLError = GraphQLError
   }
   deriving (Show, Eq, Generic, ToJSON, FromJSON)
 
+displayGraphQLError :: String -> String -> GraphQLError -> String
+displayGraphQLError indentFirst indentRest exception =
```

Comment:
Nit: I don't think this needs to be completely general 🙂 Let's just inline `"* "` and `"  "`; I suspect it'll be a bit cleaner